extensions/sentry/src/UnauthorizedError.tsx

```diff
@@ -1,12 +1,14 @@
-import { Action, ActionPanel, Detail, Icon, openCommandPreferences } from "@raycast/api";
+import { Action, ActionPanel, Detail, getPreferenceValues, Icon, openCommandPreferences } from "@raycast/api";
+const { url } = getPreferenceValues();
```

Comment:
Maybe make this as a function for all commands instead with below functionality:

Could we in the code default to the default sentry URL if user somehow deletes the the URL from preferences?

Reply:
yes we certainly could do that